backend/serverless.yml

```diff
@@ -40,17 +40,38 @@ resources:
     appReviewsTable:
       Type: AWS::DynamoDB::Table
       Properties:
-        TableName: appReviewsTable-${self:provider.stage}
+        TableName: reviewsTable-${self:provider.stage}
         AttributeDefinitions:
-          - AttributeName: id
+          - AttributeName: appId
             AttributeType: S
-          - AttributeName: store
+          - AttributeName: versionDate
             AttributeType: S
+          - AttributeName: compSentiment
+            AttributeType: N
         KeySchema:
-          - AttributeName: id
+          - AttributeName: appId
             KeyType: HASH
-          - AttributeName: store
+          - AttributeName: versionDate
             KeyType: RANGE
         ProvisionedThroughput:
           ReadCapacityUnits: 5
           WriteCapacityUnits: 5
+        GlobalSecondaryIndexes:
+          -
```

Comment:
Is there a reason there are extra newlines here?

Reply:
No, edited to match the rest of the file